### DISCUSSION TITLE
getValues ​​on nested structure

### DISCUSSION THREAD

**Discussion by lidoravitan** [react-hook-form/react-hook-form#2465] [Q&A] [answered]
Hi,
I'm using react-hooks-from for nested structure form (name="parent.child").
However, i want to use  `getValues` for nested input.
on [getValues](https://react-hook-form.com/api#getValues) section I saw there are three ways:
``` javascript
getValues() // Read all form values.
getValues('test') // Read an individual field value by name.
getValues(['test', 'test1']) // Read multiple fields by name.
```

is there any way to get the value of nested field? 
i.e `getValues('parent.child')`

**Comment by bluebill1049** [comment, marked as answer]
`getValues('parent.child')` doesn't work?

**Reply by lidoravitan** [reply]
it works! were some issue with my `Controller` implementation
